src/data/quot.lean

```diff
@@ -138,6 +147,15 @@ variables {φ : quotient sa → quotient sb → Sort*}
 
 instance [inhabited α] : inhabited (quotient sa) := ⟨⟦default⟧⟩
 
+instance (s : setoid α) [subsingleton α] : subsingleton (quotient s) :=
+begin
+  apply subsingleton.intro,
+  intros x y,
+  apply x.induction_on, intro x,
+  apply y.induction_on, intro y,
+  congr,
+end
+
```

Comment:
```suggestion
instance (s : setoid α) [subsingleton α] : subsingleton (quotient s) :=
quot.subsingleton

```
You can use the `quot` version to prove the `quotient` version. The instance gets automatically named so I can refer to it - you're also welcome to name it yourself if you prefer but that's up to you!